client/components/AllProducts.js

```diff
@@ -43,12 +43,14 @@ class AllProducts extends React.Component {
             ? filteredProducts.map(product =>
               (<li key={product.id}>
                 <Link to={`/products/${product.id}`}>{product.title}</Link>
-                <button onClick={() => this.onAddToCartClick(product)}>ADD TO CART</button>
               </li>))
             : products && products.map(product =>
               (<li key={product.id}>
+
                 <Link to={`/products/${product.id}`}>{product.title}</Link>
-                <button onClick={() => this.onAddToCartClick(product)}>ADD TO CART</button>
+                <br />
+                <img className="mini" src={  product.image } />
```

Comment:
I believe you don't need a class to set the width of the image. You could add an attribute `width="200"` and it should give you the same result.